app/src/main/res/layout/main_connection.xml

```diff
@@ -34,6 +34,13 @@
         android:visibility="gone"
         tools:visibility="visible">
 
+        <TextView
```

Comment:
Please add the test for new `TextView`.

Reply:
ok, but it's not clear how I can test. 

Since this TextView is unconditionally always visible, i'm not sure what to test 🤔 Would you have an example of a similar test for another TextView?

Reply:
https://github.com/VREMSoftwareDevelopment/WiFiAnalyzer/blob/f42bf38f61a2637672d14d2ba532375a84b31994/app/src/test/kotlin/com/vrem/wifianalyzer/wifi/accesspoint/ConnectionViewTest.kt#L108-127

Reply:
I added a visibility test: 1d16ef1

like this..?